cloudsync_onedrive.py

```diff
@@ -1106,17 +1109,19 @@ def rename(self, oid, path):  # pylint: disable=too-many-locals, too-many-branch
     def _parse_time(time_str):
         try:
             if time_str:
-                ret_val = arrow.get(time_str).timestamp
+                ret_val = arrow.get(time_str).int_timestamp
```

Comment:
Can you provide a little background on this change? thx

Reply:
this was unrelated to the rest of the changes -- it resolves a deprecation warning from the arrow lib, and should make upgrading to arrow 1.0 easier in the future.